Makefile

```diff
@@ -192,10 +192,10 @@ release:
 	hack/release.sh
 
 vendor-api:
-	cd api && go mod tidy && go mod vendor
+	cd api && go mod tidy -compat=1.17 && go mod vendor
```

Comment:
maybe worth to have ENV_VAR for the 1.17 here and in the next line ?

Reply:
Reading it from go.mod

Reply:
so it means that once go.mod is changed this need to be changed as well
maybe at least a comment on go.mod then ?